joy_teleop crashes when publishing a message with header

On ROS2 foxy, joy_teleop fails to convert `rclpy.Time` to `builtin_interfaces.msg.Time` when setting the header stamp.

```
...
  File "/opt/ros/foxy/lib/python3.8/site-packages/joy_teleop/joy_teleop.py", line 402, in joy_callback
    command.run(self, msg)
  File "/opt/ros/foxy/lib/python3.8/site-packages/joy_teleop/joy_teleop.py", line 235, in run
    msg.header.stamp = node.get_clock().now()
  File "/opt/ros/foxy/lib/python3.8/site-packages/std_msgs/msg/_header.py", line 131, in stamp
    assert \
AssertionError: The 'stamp' field must be a sub message of type 'Time'
```

--- a/joy_teleop/joy_teleop/joy_teleop.py
+++ b/joy_teleop/joy_teleop/joy_teleop.py
@@ -232,7 +232,7 @@ class JoyTeleopTopicCommand(JoyTeleopCommand):
 
         # If there is a stamp field, fill it with now().
         if hasattr(msg, 'header'):
-            msg.header.stamp = node.get_clock().now()
+            msg.header.stamp = node.get_clock().now().to_msg()
 
         self.pub.publish(msg)
 

